exporter/exporterhelper/common_test.go

```diff
@@ -20,7 +20,8 @@ import (
 
 	"github.com/stretchr/testify/require"
 	"go.opencensus.io/tag"
```

Comment:
What about this one?

Reply:
That is not tracing. Tags in opencensus are more related to metrics and cannot change to baggage until changing to otel metrics as well

Reply:
Alright, wasn't sure if it was used in the context of tracing or not.